_posts/2021-12-31-trino-2021-a-year-of-growth.md

```diff
@@ -0,0 +1,348 @@
+---
+layout: post
+title:  "Trino 2021 Wrapped: A Year of Growth"
+author: Brian Olsen, Martin Traverso, Manfred Moser
+excerpt_separator: <!--more-->
+---
+
+As we reflect on Trino’s 2021 wrapped, one thing stands out. Compared to 
+previous years we have seen even further accelerated, tremendous growth. Yes,
+this is what all these year-in-retrospect blog posts say, but this has some 
+special significance to it. This week marked the one-year anniversary since the 
+project [dropped the Presto name and moved to the Trino name]({{site.url}}/blog/2020/12/27/announcing-trino.html).
+
+## Renaming from PrestoSQL to Trino
+
+This was a gut wrenching decision for many in the community, who had grown with
+the project from early days. It also meant that the community would have to
+build affinity towards the new name from scratch. We weren't sure what to expect
+when we announced the rename. What followed, was an outpouring of support and
+shock from the larger developer community. 
+
+<p align="center">
+ <a href="https://twitter.com/trinodb/status/1343330429684703232?s=20" target="_blank">
+   <img align="center" width="100%" src="/assets/blog/2021-review/tweets.png"/>
+ </a>
+</p>
+
+<!--more-->
+
+The community immediately got to work. The project had to change the namespace
+practically overnight from the `io.prestosql` namespace to `io.trino` and
+a [migration blog post]({{site.url}}/blog/2021/01/04/migrating-from-prestosql-to-trino.html)
```

Comment:
This needs to be exposed more prominently. Specifically, where to find the code, slack, docker images, etc. Some people are still getting confused or lost.